libctru/include/3ds/services/frd.h

```diff
@@ -4,6 +4,7 @@
  */
 #pragma once
 #include <3ds.h>
```

Comment:
I just realized that 3ds.h is also included from frd.h. Can you fix this so that the actual headers required by frd.h are included?

Reply:
Fixed + mcuhwc.c was having the same issue, fixed that too.